java/Mekhmet-Efe_Adzhar/src/com/zoolatech/lecture5/tasks/_4/Number4.java

```diff
@@ -0,0 +1,50 @@
+package com.zoolatech.lecture5.tasks._4;
+
+/*
+Create a method that accepts a string, counts all unique non-space character occurrences and displays each character
+with an amount of occurrences (output should be sorted by a character value in ascending order).
+Also, add a couple of examples that use your method.
+Input: “Hello World”
+Output:
+H - 1
+W - 1
+d - 1
+e - 1
+l - 3
+o - 2
+r - 1
+ */
+
+import java.util.Scanner;
+
+public class Number4 {
+    public static void main(String[] args) {
+        //  String string = "Hello World";
+        Number4 number4 = new Number4();
+        //number4.letterCounter(string);
+        Scanner scanner = new Scanner(System.in);
+        System.out.println("Type a String");
+        String secondString = scanner.nextLine();
+        number4.letterCounter(secondString);
+
+    }
+
+    public void letterCounter(String string) {
+        int[] ints = new int[string.length()];
+        char[] chars = string.toCharArray();
+
+        for (int i = 0; i < string.length(); i++) {
```

Comment:
try to find a solution which uses only a single loop (hint: you can use maps for this)